main.py

```diff
@@ -79,4 +79,24 @@ async def IsMemberWorking(ctx, discorduser, date = datetime.today().strftime('%d
 
     await ctx.send(content = "Done", delete_after = 0.1)
 
+@slash.slash(name="IsEveryoneHere", description="Checks if all working 'Full Time' members are in voice channel", guild_ids=guild_ids)
+async def IsEveryoneHere(ctx):
+
+    guild = client.guilds[0]
+    voice_channel = client.get_channel(int(os.getenv("MeetingChannel_id")))
+    fulltime_role = discord.utils.get(guild.roles, name = "Full Time")
+    fulltime_member_list = list(filter(lambda user: fulltime_role in user.roles, guild.members))
+    voicechannel_fulltime_member_list = list(filter(lambda user : fulltime_role in user.roles, voice_channel.members))
+
+    not_here = list(set(fulltime_member_list) - set(voicechannel_fulltime_member_list))
+    not_here = list(filter(lambda user : (leave_interface.IsMemberWorking(user.id, datetime.today()))[0], not_here))
```

Comment:
Can you create a card in to do to deal with departments that don't need to attend core please, when that's done, link that here.

Reply:
Card added